content/monad-transformers.md

```diff
@@ -23,13 +23,96 @@ A `Reader r a` in which the resulting `a` is wrapped in any other monad, i.e. `r
 
 ### StateT
 
-A `State s a` in which the return value `a` and state `s` are wrapped in any other monad, i.e. `s -> m (a, s)`
+A `State s a` in which the return value and state, namely `(a, s)`, are wrapped in any other monad, i.e. `s -> m (a, s)`
 
-### EitherT
```

Comment:
I actually would stick with the `EitherT`. I discussed it with others some time in the past, and there are still ways in which `EitherT` is superior.